distro/exporter.go

```diff
@@ -298,3 +320,28 @@ func noneEnvVarSet(envs ...string) bool {
 func notNone(s string) bool {
 	return s != "" && s != "none"
 }
+
+//nolint:unparam // This will receive other input values in future.
+func otlpProtocol(l logr.Logger, signalKey string) string {
+	// Signal-specific key takes precedence.
+	if v := os.Getenv(signalKey); v != "" {
```

Comment:
Should we support values like `"gRPC"` as well?

Reply:
Fine. I will create an issue for it here an work on the spec level to allow it.